src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -175,6 +175,64 @@ def test_backup_container(self, resource_group, vault_name, vm1, vm2):
             self.check("length([?properties.friendlyName == '{vm1}'])", 1),
             self.check("length([?properties.friendlyName == '{vm2}'])", 1)])
 
+    def test_backup_wl_container(self, container_name1='shrac3', container_name2='shrac2', resource_group='shracrg', vault_name='shracsql'):
+
+        self.kwargs.update({
+            'vault': vault_name,
+            'name1': container_name1,
+            'name2': container_name2,
+            'rg': resource_group,
+            'rvault': 'pstestwlRSV1bca8',
+            'rrg': 'pstestwlRG1bca8',
```

Comment:
rename it to vault rg or rg vault, something on those lines to make it intuitive

Reply:
Does not exist in current commit.